app/knowledge_panels.py

```diff
@@ -39,3 +41,49 @@ def hunger_game_kp(
             ],
         },
     }
+
+
+def data_quality_kp(
+    facet,
+    value: Union[str, None] = None,
+    country: Union[str, None] = None,
+):
+    if facet == "country":
+        path = f"{facet}/{value}"
+        description = f"{value}"
+    else:
+        if country is not None:
+            path = f"country/{country}/{facet}/{value}"
+            description = f"{facet} based for {country}"
+        else:
+            path = f"{facet}/{value}"
+            description = facet
+    url = "https://world.openfoodfacts.org/"
+    source_url = urljoin(url, path)
+    description = f"data-quality issues related to {description}"
+    html = f"{source_url}/data-quality.json"
+    """
+    Parsing data from the url
+    """
+    response_API = requests.get(html)
+    data = response_API.text
+    parse_json = json.loads(data)
```

Comment:
maybe ?
```suggestion
    data = response_API.json()
```

Reply:
Its throw type error, Is this important?